bundle/manifests/netobserv-operator.clusterserviceversion.yaml

```diff
@@ -203,7 +203,7 @@ metadata:
                   },
                   "enable": false
                 },
-                "url": "http://loki.netobserv.svc:3100/"
+                "url": "http://loki.netobserv.svc.cluster.local.:3100/"
```

Comment:
There's an extra "." before ":3100".  This one is critical, since this is a YAML file unlike before which were just documents.

It seems this issue is in many other places so correct them all.